guardian/archivehunter
# Fix/autodowing lambda

## What does this change?

- Updates the autodowning lambda code to AWS SDK v2 
- Thins down its dependencies so that it deploys properly again
- Also includes library fixes to allow the kinesis logger to start up properly again; this was causing a deployment failure in the autoscale group

## How to test

- RiffRaff deployment should not longer fail
- Once deployment has been done once, deploy again and check the Cloudwatch logs for the autodowning lambda.  You should see it functioning normally - detecting the new instances and storing them in Dynamo then detecting the shut-down instances and removing them from the cluster

## How can we measure success?

RiffRaff deployments not erroring any more
